lib/ansible/modules/cloud/vmware/vmware_drs_group.py

```diff
@@ -0,0 +1,600 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Karsten Kaj Jakobsen <kj@patientsky.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+author:
+  - "Karsten Kaj Jakobsen (@karstenjakobsen)"
+description:
+  - "This module can be used to create VM/Host groups in a given cluster. Creates a vm group if C(vms) is set. Creates a host group if C(hosts) is set."
+extends_documentation_fragment: vmware.documentation
+module: vmware_drs_group
+notes:
+  - "Tested on vSphere 6.5 and 6.7"
+options:
+  cluster_name:
+    description:
+      - "Cluster to create vm/host group."
+    required: true
+  datacenter:
+    aliases:
+      - datacenter_name
+    description:
+      - "Datacenter to search for given cluster. If not set, we use first cluster we encounter with C(cluster_name)."
+    required: false
+  group_name:
+    description:
+      - "The name of the group to create or remove."
+    required: true
+  hosts:
+    description:
+      - "List of hosts to create in group."
+      - "Required only if C(vms) is not set."
+    required: false
+  state:
+    choices:
+      - present
+      - absent
+    default: present
+    description:
+      - "If set to C(present) and the group doesn't exists then the group will be created."
+      - "If set to C(absent) and the group exists then the groupwill be deleted."
+    required: true
+  vms:
+    description:
+      - "List of vms to create in group."
+      - "Required only if C(hosts) is not set."
+    required: false
+requirements:
+  - "python >= 2.6"
+  - PyVmomi
+short_description: "Creates vm/host group in a given cluster."
+version_added: "2.8"
+'''
+
+EXAMPLES = r'''
+---
+- name: "Create DRS VM group"
+  delegate_to: localhost
+  state: present
```

Comment:
`state` should be in the module parameters
```suggestion
  vmware_drs_group:
      state: present
```